ktor-client/ktor-client-core/js/src/io/ktor/client/features/websocket/JsWebSocketSession.kt

```diff
@@ -109,6 +112,11 @@ internal class JsWebSocketSession(
         }
     }
 
+    @OptIn(ExperimentalWebSocketExtensionApi::class)
+    override fun start(negotiatedExtensions: List<WebSocketExtension<*>>) {
+        require(negotiatedExtensions.isEmpty())
```

Comment:
Maybe add some message here and in similar places?

Reply:
Sure will add